zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## 0.5.0
+
+### Features
+
+- Z-Wave JS: Map more Basic CC values to more useful command classes

```

Comment:
Slightly obfuscated in the changelog, but 12.5.4 also fixes an issue where spurious Basic (and other) values could show up when they shouldn't.